src/openrct2-ui/scripting/ScWidget.hpp

```diff
@@ -969,7 +969,8 @@ namespace OpenRCT2::Scripting
             auto w = GetWindow();
             if (w != nullptr && IsCustomWindow())
             {
-                w->OnMouseDown(_widgetIndex);
+                WindowStartTextbox(
+                    *w, _widgetIndex, GetWidget()->string, OpenRCT2::Ui::Windows::GetWidgetMaxLength(w, _widgetIndex));
```

Comment:
ScWidget is defined in the OpenRCT2 namespace, so `Ui::Windows::GetWidgetMaxLength` should suffice. As a bonus, that likely also makes things fit on one line again.

Reply:
> ScWidget is defined in the OpenRCT2 namespace, so `Ui::Windows::GetWidgetMaxLength` should suffice. As a bonus, that likely also makes things fit on one line again.

There are multiple other instances in this file where `OpenRCT2::Ui::Windows::` is used. Should I make the style consistent here?

Reply:
Hmm, I'd prefer this PR to only concern itself with introducing the new feature. If you like, I'd welcome a PR removing superfluous namespace resolution for the project in general, though.